### ISSUE TITLE
do not include a tag in the installed plugin

### ISSUE THREAD

**Issue by masuhar** [prg-titech/bib-shortcode#9]
Currently the plugin is installed in  `public_html/wp-content/plugins/bib-shortcode-0.1.1-alpha` at the web server, where the tag part (`-0.1.1-alpha`) should not be included.

- [x] check the zip file name and the contents
- [x] check how Wordpress determines the installation path
- [ ] find out a solution

**Comment by masuhar** [comment]
The file name and the contents of the .zip file:
```
Archive:  bib-shortcode-0.1.1-alpha.zip
Zip file size: 2023 bytes, number of entries: 4
drwx---     0.0 fat        0 bx stor 24-Aug-17 07:03 bib-shortcode-0.1.1-alpha/
-rw----     0.0 fat      228 tx defN 24-Aug-17 07:03 bib-shortcode-0.1.1-alpha/Rakefile
-rw----     0.0 fat     1334 tx defN 24-Aug-17 07:03 bib-shortcode-0.1.1-alpha/prg-bib.php
-rw----     0.0 fat      769 tx defN 24-Aug-17 07:03 bib-shortcode-0.1.1-alpha/readme.txt
4 files, 2331 bytes uncompressed, 1319 bytes compressed:  43.4%
```

**Comment by masuhar** [comment]
> check how Wordpress determines the installation path

No descriptions are found in https://developer.wordpress.org/plugins/plugin-basics/, but let's assume that WordPress uses the directory name inside the .zip file, not the .zip file name.

For example, the classic-editor plugin has the following .zip file name and the content file names.  This backs up the above assumption.
```
Archive:  classic-editor.1.6.4.zip
Zip file size: 19273 bytes, number of entries: 6
drwxr-xr-x  3.0 unx        0 b- stor 24-Jul-19 18:20 classic-editor/
-rw-r--r--  3.0 unx    36305 t- defN 24-Jul-19 18:20 classic-editor/classic-editor.php
drwxr-xr-x  3.0 unx        0 b- stor 24-Jul-19 18:20 classic-editor/js/
-rw-r--r--  3.0 unx      624 t- defN 20-Aug-11 20:12 classic-editor/js/block-editor-plugin.js
-rw-r--r--  3.0 unx    19104 t- defN 20-Aug-11 20:12 classic-editor/LICENSE.md
-rw-r--r--  3.0 unx     7768 t- defN 24-Jul-19 18:07 classic-editor/readme.txt
6 files, 63801 bytes uncompressed, 18483 bytes compressed:  71.0%
```

**Comment by masuhar** [comment]
[Github doesn't provide an option to change the folder names.](https://stackoverflow.com/questions/68794471/how-to-change-the-github-zip-releases-folder-names#answer-68795192)

**Comment by masuhar** [comment]
- Solution 1: write a Rakefile that generates a .zip file, and upload the file manually.
- Solution 2: let CI do the above.

**Comment by masuhar** [comment]
https://asukiaaa.blogspot.com/2023/04/create-zip-and-show-info-on-release-body-with-using-github-actions.html

**Comment by masuhar** [comment]
resolved in 9d7260b83a9550ffbdb56ad61880ee1991f0f5e8
